src/pages/CatIndex.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+class CatIndex extends Component {
+    render() {
+        return (
+            <>
+            <h2>CatIndex</h2>
```

Comment:
If you are going to have multiple elements remember to indent.

Reply:
resolved